CHANGELOG.md

```diff
@@ -11,6 +11,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
+### Removed
+
+# 0.16.2 - 2021-03-22
+
+### Changed
```

Comment:
What's the rule in the changelog around the "Added", "Changed" & "Removed" headings for each update? Some updates seem to have all three regardless of whether all of them have any content, and some updates only have headings (like this one) if there is content.

Only reason I ask is the headings + GitLens in VSCode makes the file look very noisy:
![Screenshot from 2021-03-22 11-06-30](https://user-images.githubusercontent.com/2376968/111980849-c8325480-8afe-11eb-9729-9ca00d53b686.png)

Reply:
Not sure if there is a rule around that. But I guess it makes sense only to keep the ones that have any items.